docs/quickstart.ipynb

```diff
@@ -1794,7 +1796,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "item_to_update = root_catalog.get_item(\"LC80140332018166LGN00\", recursive=True)\n",
+    "item_to_update = next(\n",
+    "    root_catalog.get_items(\"LC80140332018166LGN00\", recursive=True), None\n",
+    ")\n",
```

Comment:
```suggestion
    "item_to_update = next(\n",
    "    root_catalog.get_items(\"LC80140332018166LGN00\", recursive=True)\n",
    ")\n",
```

Same here, we don't check to see if `item_to_update` is `None`, so may make more sense to allow the exception through.